system-contracts/contracts/interfaces/IL2GenesisUpgrade.sol

```diff
@@ -2,12 +2,32 @@
 // We use a floating point pragma here so it can be used within other projects that interact with the ZKsync ecosystem without using our exact pragma version.
 pragma solidity ^0.8.20;
 
+struct AdditionalForceDeploymentsData {
```

Comment:
Maybe better to call it `ZKChainSpecificForceDeploymentsData`? And also we need to add a comment on the difference between those